### PR TITLE
Improve and fix texts of confirmation popups in UI

### PR DESCRIPTION
**Pull request by StefanKlt**
Fix text in popup showed when turning a soft into a force action. Add plural 's' when assigning multiple entities. Fix i18n for UI event notification.

Signed-off-by: Stefan Klotz <stefan.klotz@bosch-si.com>

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/event/EntityModifiedEventPayloadIdentifier.java [hunk 1]
```diff
@@ -23,15 +22,15 @@
     private final Class<? extends ProxyIdentifiableEntity> parentType;
     private final Class<? extends ProxyIdentifiableEntity> entityType;
     private final EntityModifiedEventType modifiedEventType;
-    private final String eventTypeMessageKey;
+    private final EventNotificationType notificationType;
```
Comment (bogdan-bondar) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/event/EntityModifiedEventPayloadIdentifier.java:25]: Could you please adapt the comment under `#hashCode` and `#equals` that we ignore notificationType

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/grid/support/assignment/DeploymentAssignmentSupport.java [hunk 2]
```diff
@@ -73,7 +73,9 @@ private ConfirmationDialog createConfirmationWindow(final String confirmationMes
         }, content, confirmationWindowId());
     }
 
-    protected abstract String sourceEntityType();
+    protected abstract String sourceEntityTypeOne();
```
Comment (bogdan-bondar) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/grid/support/assignment/DeploymentAssignmentSupport.java:76]: Could you use sourceEntityTypeSing and sourceEntityTypePlur to be consistent

**Review by bogdan-bondar — CHANGES_REQUESTED** [review]
Great Job!

**Comment by stefbehl** [comment]
@hawkbit-bot verify please

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/event/EntityModifiedEventPayloadIdentifier.java (hunk 1 above)
Comment (StefanKlt) [reply hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/event/EntityModifiedEventPayloadIdentifier.java:25]: done

**Review by bogdan-bondar — APPROVED** [review]
Thanks a lot, looks nice!

**Pull request merged**